src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAllAatfReuseSitesCsvDataTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace EA.Weee.DataAccess.Tests.DataAccess.StoredProcedure
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Domain.AatfReturn;
+    using FluentAssertions;
+    using Weee.Tests.Core;
+    using Weee.Tests.Core.Model;
+    using Xunit;
+    using Return = Domain.AatfReturn.Return;
+    using ReturnReportOn = Domain.AatfReturn.ReturnReportOn;
+    using WeeeReusedSite = Domain.AatfReturn.WeeeReusedSite;
+
+    public class GetAllAatfReuseSitesCsvDataTests
+    {
+        [Fact]
+        public async Task Execute_GivenNoData_NoResultsShouldBeReturned()
+        {
+            using (var db = new DatabaseWrapper())
+            {
+                var @return = ObligatedWeeeIntegrationCommon.CreateReturn(null, db.Model.AspNetUsers.First().Id, FacilityType.Aatf);
```

Comment:
Can a test be added to also check the that field values are expected as well as the count... i.e. that the AatfReuseSitesData is populated as expected

Reply:
Updated the test